ui/src/app/shared/services/accounts-service.ts

```diff
@@ -25,7 +25,9 @@ export class AccountsService {
     }
 
     public deleteToken(name: string, id: string): Promise<any> {
-        return requests.delete(`/account/${name}/token/${id}`);
```

Comment:
I think it is safe to add header here for all requests: https://github.com/argoproj/argo-cd/blob/32e373829ba3e69dea216056c80f490ecd2c6264/ui/src/app/shared/services/requests.ts#L39